### DISCUSSION TITLE
SaltEdge integrations smaller than expected

### DISCUSSION THREAD

**Discussion by Dagnan** [firefly-iii/firefly-iii#8676] [General]
Hi all.
When I contacted SaltEdge as part of the onboarding process, here is what they told me:

> Before proceeding, I would like to let you know that the partnership with FireFly III does not cover the possibility of connecting financial institutions integrated via PSD2/Open Banking channels in the EU & UK, thus the only available connections are powered by Web Scraping technologies.
> 
> To review the list of financial institutions connected with this type of technology, please follow [this link to see the available banks](https://www.saltedge.com/products/account_information/coverage/?channel=non_regulated).

For France, it means it goes from 120 integrations to 3. Germany is well served with 2k+. Just to be extra clear, those integrations are done through scrapping and not APIs.

I feel  like this should be mentioned somewhere and would save people a bit of time.

Should I open a pull-request?

**Comment by JC5** [comment]
What did you have in mind?

**Comment by Dagnan** [comment]
TBH, after I wrote this, I realized it was already in the documentation. Yet, I did miss it there.

I think mentioning it in the README would be good. Something like: "EU citizens: you might want to use Nodigen/GoCardLess instead of SaltEdge. The latter has only a few integrations in EU which only rely on scrapping".
